### ISSUE TITLE
Add accessibility into OpenTX Companion

### ISSUE THREAD

**Issue by bobaustralia** [opentx/opentx#2792]
HI, OPET TX. Not sure if this is an issue or not what I am asking if it is possible is to have it access accessibility software inserted in future versions of  OPET TX access accessibility is a free program produced by Microsoft for screen readers. I suffer from dyslexia and I use a screen reader program called read and write by text help on my computer. I have been flying model planes was several years and have used a spectrum DX7 radio but I want now upgrade to Tirana’s and if this program was in OPET TX will allow me to use the program by myself instead of having to asked for help other club members.
Bob.

**Comment by projectkk2glider** [comment]
I always thought that the accessibility was provided by the operating system itself. Or does each program (in our case OpenTX Companion) have to have a special support for this kind of features?

**Comment by bobaustralia** [comment]
Hi. The active accessibility has to be embedded in the software itself for the screen reader program to work US software is required by federal law to have it however most European software doesn’t have though I have an OCR program that is made by a Russian company it has active accessibility installed in it I hope this is helped.

 

Regards, Bob.

**Comment by bsongis** [comment]
I was not aware of that, I will see if Qt does provide an easy way to bring
to Companion the accessibility feature

**Comment by bobaustralia** [comment]
Hi. One other thing I meant to mention to the forum was that open pilot software has active accessibility built-in as downloaded it and mucking around with it before wanting to buy a quad with  CC3D may be?.

 

Regards, Bob.

 

PS. Anybody’s wondering these posts are dictated using dragon 12.

**Comment by bsongis** [comment]
I did some commits in OP. Good to know accessibility is implemented there,
they also use Qt!
